scripts/emulations/README.md

```diff
@@ -0,0 +1,23 @@
+# Add Validator
+
+This script will help us in emulating various feature in blockchain programmatically. 
```

Comment:
This script emulates various features in the Cere Network.

Reply:
Updated.